app/routes/goal_routes.py

```diff
@@ -1 +1,93 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from ..routes.utilities import create_model
+from ..models.goal import Goal
+from ..db import db
+from ..routes.utilities import validate_model
+from ..models.task import Task
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+@bp.post("")
+def creaet_a_goal():
+    request_body = request.get_json()
+    if request_body == {}:
+        response = {"details": "Invalid data"}
+        return response, 400
+
+    return {"goal": create_model(Goal, request_body)}, 201
+
+
+@bp.get("")
+def get_all_goals():
+    query = db.select(Goal).order_by(Goal.id)
+    goals = db.session.scalars(query)
+
+    response = []
+
+    for goal in goals:
+        response.append(goal.to_dict())
+    return response
```

Comment:
Make sure to use whitespace to distinguish different logic blocks.
```suggestion
    for goal in goals:
        response.append(goal.to_dict())
        
    return response
```